src/main/java/TaskList.java

```diff
@@ -0,0 +1,53 @@
+import java.util.ArrayList;
+
+/**
+* Add the ability to store whatever text entered by the user and display them back to the user when requested.
+* */
+
+public class TaskList {
+    private ArrayList<Task> taskList = new ArrayList<Task>();
+
+    // constructors
+
+    public TaskList(){
+    }
+
+    public String getTaskDescription(int serialNumber){
+        String taskDescriptionOutput = "  [";
+        if (taskList.get(serialNumber - 1).getIsDone()){
+            taskDescriptionOutput += "X] ";
```

Comment:
A bit confused at first sight

Reply:
I'll try to make the indexing clearer and remove some potential magic numbers, thanks!